code/dictation.py

```diff
@@ -23,33 +38,45 @@ def word(m) -> str:
 @mod.capture(rule="({user.vocabulary} | <phrase>)+")
 def text(m) -> str:
     """A sequence of words, including user-defined vocabulary."""
-    return format_phrase(m)
+    return apply_formatting(m)
 
-@mod.capture(rule="({user.vocabulary} | {user.punctuation} | <phrase>)+")
+@mod.capture(rule="({user.vocabulary} | {user.punctuation} | {user.prose_snippets} | <phrase> | {user.prose_modifiers})+")
 def prose(m) -> str:
     """Mixed words and punctuation, auto-spaced & capitalized."""
-    text, _state = auto_capitalize(format_phrase(m))
-    return text
+    return apply_formatting(m).replace("“", "\"").replace("”", "\"")
```

Comment:
Yes, that's exactly why I'm doing it. I agree this is non-obvious and I'm happy to document if we decide to keep this in. Personally I don't want to output unstraightened quotes -- I can't do this with my keyboard so I'm fine not having that with voice, and it can cause trouble if they make it into code. But I would like to make this work for folks like yourself if you find it useful. I'm not sure there is a both clean and simple way to do this, although we can use any characters (or sequence of characters) we want, so we could do double curly quotes or something like that (we'd need to fix the regexes).

Reply:
I was thinking we could bind "open quote" to `' "'` (double quote with a space before it) and close quote to `'" '` (double quote with a space after) work? Unfortunately, this produces spurious spaces at the beginning/end of documents.

I'll have to think more about this problem. But I think your use-case is more common/important than mine - most people want straight quotes and not curly. So if push comes to shove I'm inclined to accept your approach on this; we can always fix it up with another PR later.

Reply:
We could add a "straighten_quotes" setting and check that in the two places that straightening is done in this file. This also lets people set it per-app, which I could imagine some folks wanting.

Reply:
I think it's better to leave the PR as-is rather than adding a setting. I have an idea for how to get the spacing right without using curly quotes and then straightening them, which would obviate the need for a setting. I'll make an issue for it shortly.